buildtools/bundle.js

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env node
+
+/**
+ *
+ * This is a script to bundle up some widgets and their
+ * assets into a standalone directory that could be used
+ * offline. It is a little bit string and duct-tape-y,
+ * sorry about that.
+ *
+ * Requires wget to operate. Will only bundle widgets
+ * that have an "archive" field in their package.json.
+ * The "archive" field can be an empty {}, or:
+ *    {
+ *      "domains": ["domain1.com", "domain2.com"],
+ *      "entrypoints: ["https://other.page"]
+ *    }
+ * Material in the specified domains will also be
+ * included in the bundle. Material needed by any
+ * extra entry-point pages will be included as well.
+ *
+ * The bundle will also include grist-plugin-api.js.
```

Comment:
This doesn't work for me, should it be added to the domains?

Reply:
Ah, sorry, I had commented out a `// domains.push('getgrist.com')` line. I put it back in.